README.md

```diff
@@ -0,0 +1,6 @@
+# Sinhala-Word-Embeddings
```

Comment:
Please add Numpy compatible docstrings to module/class and methods
e.g. https://www.jetbrains.com/help/pycharm/settings-tools-python-integrated-tools.html
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring